client/my-sites/patterns/components/readymade-template-details/server.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReadymadeTemplateDetails } from 'calypso/my-sites/patterns/components/readymade-template-details/index';
+
+export const ReadymadeTemplateDetailsServer = ( { id }: { id: number } ) => {
```

Comment:
I think you can use the `ReadymadeTemplateDetailsFC` type here.

Reply:
Never mind, just seen `ReadymadeTemplateDetailsFC` is now used for the inner component.